packages/file_selector/file_selector_windows/pubspec.yaml

```diff
@@ -23,3 +23,8 @@ dependencies:
     sdk: flutter
   flutter_test:
     sdk: flutter
+
+dev_dependencies:
+  build_runner: 2.1.11
+  mockito: ^5.1.0
+  pigeon: ^3.1.4 # DO NOT LAND: Needs updating to actual version once published
```

Comment:
This PR relies on https://github.com/flutter/packages/pull/2270 which is why this has a comment; I'll land that first, but wanted to get the review going in parallel.